android/app/src/main/java/nem/group/wallet/MainActivity.java

```diff
@@ -1,4 +1,4 @@
-package nem.group.wallet;
+package nem.group.symbol.wallet;
```

Comment:
I don't think we need to change the package name but well, since you did we can keep it.

Reply:
But `nem.group.symbol.wallet` is used for play store right?

Reply:
I see, so it is not related to the package itself.
i will update it :)